### ISSUE TITLE
QR Scanner not working in release mode

### ISSUE THREAD

**Issue by priyansh360** [EddyVerbruggen/nativescript-barcodescanner#241]
Hi , 
Can you please help me out in my issue , I added this plugin to my NS-ng app it is working fine while in development mode but when I generate a signed APK/release in both iOS & Android the plugins doesn't seems to be work.
Please reply

**Comment by Rdevc** [comment]
Hi,

This is probably to late for you but I hope to help others responding on this issue. 

After several hours of debugging and trying I figured out the issue and it was possible to debug trough the iOS emulator. 

**Solution:**
- Delete `beepOnScan="false"` or `beepOnScan="true` from the html
- Set in the new barcodescanner().scan the beepOnScan to false  like `beepOnScan: false` now your scanner will be working again outside of development mode. 

@EddyVerbruggen; 
When hitting the code with beepOnScan: true the following error will occur "AVFAudio`-[AVAudioPlayer initWithContentsOfURL:fileTypeHint:error:]:" with "Thread 1: EXC_BAD_ACCESS (code=1, address=0x18)"

I'm 90% sure one of these lines of code are responsible:
```
  line 385: var barcodeBundlePath = NSBundle.bundleWithIdentifier("com.telerik.BarcodeScannerFramework").bundlePath;
  line 386 this._player = new AVAudioPlayer({ contentsOfURL: NSURL.fileURLWithPath(barcodeBundlePath + "/beep.caf") });
```

Kind regards,
R

**Comment by EddyVerbruggen** [comment]
Fixed in the (just published) 4.1.0 version.
